MIGRATING_v2_to_v3.md

```diff
@@ -0,0 +1,236 @@
+# Okta Java Management SDK Migration Guide
```

Comment:
Use the existing MIGRATING.md file, just add a new section to the top of the file

Reply:
File updated